.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+jobs:
+  build:
+    machine:
+      enabled: true
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - run:
+          name: Copy env file
+          command: cp .env.dist .env
+      - run:
+          name: Setup environment
+          command: ./bin/docker/pim-setup.sh
+      - run:
+          name: Change owner on project dir (default user = circleci (1001) and docker needs uid 1000)
+          command: sudo chown -R 1000:1000 ../project
+      - run:
+          name: Start containers
+          command: docker-compose up -d
+      - run:
+          name: Install dependencies
+          command: ./bin/docker/pim-dependencies.sh
+      - run:
+          name: Pim installation
+          command: |
+              docker-compose exec fpm bin/console --env=prod pim:install --force --clean
+              docker-compose run --rm node yarn run webpack
```

Comment:
```suggestion
            &&  docker-compose run --rm node yarn run webpack
```

Reply:
why?

Reply:
Why doing this? for blocking in case of error?

Reply:
yes, it's the goal of this build actually ;)

Reply:
Ok, fine so ;) Otherwise I didn't understood :D